dotcom-rendering/src/components/EditorialButton/types.ts

```diff
@@ -0,0 +1,19 @@
+import type { ArticleFormat } from '@guardian/libs';
+
+export interface SharedEditorialButtonProps {
+	/**
+	 * A format object denoting the style of the button using the enums
+	 * available from [@guardian/libs](https://github.com/guardian/libs/blob/main/src/format.ts).
```

Comment:
This link could probably be updated to point to the local versions in `format` once #12461 is merged? Or it could use JSDoc's `linkcode`?

Reply:
Good shout, I'll make a note to do this once #12461 is merged